modules/microshift-k8s-apis.adoc

```diff
@@ -6,17 +6,4 @@
 [id="microshift-k8s-apis_{context}"]
 = {product-title} Kubernetes APIs
 
-{product-title} supports the following standard Kubernetes APIs:
-
-NOTE: TABLE IS FOR PLACEMENT ONLY (FPO)
-
-[cols="1,1",options="header"]
-|===
-^| API ^| API group
-| xref:../microshift_rest_api/network_apis/route-route-openshift-io-v1.adoc#route-route-openshift-io-v1[Route]
-| route.openshift.io/v1
-| xref:../microshift_rest_api/security_apis/securitycontextconstraints-security-openshift-io-v1.adoc#securitycontextconstraints-security-openshift-io-v1[SecurityContextConstraints]
-| security.openshift.io/v1
-|===
-
-//FIXME: Links are broken, table is incorrectly displaying
+The Kubernetes API is 100% accessible within {product-title} and can be managed with the `kubectl` command-line tool or the {OCP} CLI tool (oc). The `oc` tool is compatible with `kubectl` and offers a set of features that can be used with {product-title}. Having both tools installed for use with your {product-title} can help you access all of the resources you need to add authentication, networking, and security to your deployments.
```

Comment:
I'm not sure what "add authentication, networking, and security" means in this context. What did you have in mind for that? Would it convey the same sentiment if we just say something like "... you need to work with your deployments." instead?

Reply:
We can say that, sure.